fetch-utils.js

```diff
@@ -1,12 +1,30 @@
-const SUPABASE_URL = '';
-const SUPABASE_KEY = '';
+const SUPABASE_URL = 'https://lxifeplqjanpuugbzgyn.supabase.co';
+const SUPABASE_KEY = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imx4aWZlcGxxamFucHV1Z2J6Z3luIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NDQ0MzU0NTEsImV4cCI6MTk2MDAxMTQ1MX0.4hkIbGdYMJ7SzqMSm39JtJRvMyFal0vMXHHWB0APEJw';
 
 const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
 
 export function getUser() {
     return client.auth.session() && client.auth.session().user;
 }
 
+export async function getWorkshops() {
+    const response = await client.from('workshops').select(`*, participants (*)`);
+    console.log(response);
+    return checkError(response);
+}
+
+export async function createParticipant(data) {
```

Comment:
functions look good! I'd suggest instead of passing data as an argument make it a more descriptive/accurate word like person or participant. If you were working on a team and someone was looking this over they'd have no idea of what data was in the data object but "person" would make it more clear what argument the function will be called with.